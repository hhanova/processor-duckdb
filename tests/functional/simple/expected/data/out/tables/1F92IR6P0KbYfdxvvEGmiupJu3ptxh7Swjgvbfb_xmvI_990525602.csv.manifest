{"destination":"in.c-keboola-ex-google-drive-744153.ss-academy-project-control-sheet-session-track-mapping","incremental":false}