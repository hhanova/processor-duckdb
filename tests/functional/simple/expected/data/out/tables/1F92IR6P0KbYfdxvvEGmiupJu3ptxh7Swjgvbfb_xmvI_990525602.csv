track,session_id,session_name,timestamp_custom
product_owner,1152674,Platform Introduction (PR),2023-04-01
data_specialist,1152674,Platform Introduction (PR),2023-04-01
data_specialist,1249022,SQL Syntax Training (PR),2023-04-01
data_specialist,1152675,Best Practices (PR),2023-04-01
data_specialist,1152676,Debug Techniques (PR),2023-04-01
data_specialist,1155656,Common Components and Processors (PR),2023-04-01
sr_data_developer,1152674,Platform Introduction (PR),2023-04-01
sr_data_developer,1249022,SQL Syntax Training (PR),2023-04-01
sr_data_developer,1152675,Best Practices (PR),2023-04-01
sr_data_developer,1152676,Debug Techniques (PR),2023-04-01
sr_data_developer,1174202,Generic Extractor (PR),2023-04-01
sr_data_developer,1155656,Common Components and Processors (PR),2023-04-01
sr_data_developer,1174274,KBC APIs (PR),2023-04-01
sr_data_developer,1173117,Data Science (PR),2023-04-01
platform_developer,1152674,Platform Introduction (PR),2023-04-01
platform_developer,1249022,SQL Syntax Training (PR),2023-04-01
platform_developer,1152675,Best Practices (PR),2023-04-01
platform_developer,1152676,Debug Techniques (PR),2023-04-01
platform_developer,1174202,Generic Extractor (PR),2023-04-01
platform_developer,1174222,Project Health Check (PR),2023-04-01
platform_developer,1155656,Common Components and Processors (PR),2023-04-01
platform_developer,1174224,Component Behind the Scenes (PR),2023-04-01
platform_developer,1174264,Component Creation (PR),2023-04-01
platform_developer,1174274,KBC APIs (PR),2023-04-01
platform_developer,1173117,Data Science (PR),2023-04-01
platform_developer,1173118,Data Governance (PR),2023-04-01
data_scientist,1152674,Platform Introduction (PR),2023-04-01
data_scientist,1249022,SQL Syntax Training (PR),2023-04-01
data_scientist,1152675,Best Practices (PR),2023-04-01
data_scientist,1152676,Debug Techniques (PR),2023-04-01
data_scientist,1155656,Common Components and Processors (PR),2023-04-01
data_scientist,1173117,Data Science (PR),2023-04-01
admin,1152674,Platform Introduction (PR),2023-04-01
admin,1249022,SQL Syntax Training (PR),2023-04-01
admin,1152675,Best Practices (PR),2023-04-01
admin,1152676,Debug Techniques (PR),2023-04-01
admin,1174202,Generic Extractor (PR),2023-04-01
admin,1174222,Project Health Check (PR),2023-04-01
admin,1155656,Common Components and Processors (PR),2023-04-01
admin,1174274,KBC APIs (PR),2023-04-01
admin,1173117,Data Science (PR),2023-04-01
admin,1173118,Data Governance (PR),2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
,#N/A,,2023-04-01
